### DISCUSSION TITLE
[Suggestion] Compatibility with the WoodStuff mod

### DISCUSSION THREAD

**Discussion by xronin01** [Roadhog360/Et-Futurum-Requiem#459] [Ideas]
https://github.com/buttilda/WoodStuff

**Comment by Roadhog360** [comment]
I'm not actually sure what you are asking for here. Aren't you supposed to manually configure this stuff? What compatibility is there to add

**Reply by xronin01** [reply]
The Woodstuff mod not only adds fences and buttons, but also has chests and crafting tables that don't work with ETF planks

**Reply by Roadhog360** [reply]
> The Woodstuff mod not only adds fences and buttons, but also has chests and crafting tables that don't work with ETF planks

But again don't you have to configure that stuff yourself, my pla ks are tagged with the correct tag. If they cannot detect it automatically that means you either have to configure it manually, or they don't properly check the OreDictionary which is not my issue, but theirs.
